inspector/scripts/utils.ts

```diff
@@ -13,3 +12,76 @@ export function getKeyHash() {
     throw new Error(`Key Hash is not defined for network: ${networkName}`);
   }
 }
+
+export async function readAll(inspectorConsumer: Contract) {
+  const { sRandomWord, vrfRequestId } = await readVrf(inspectorConsumer);
+  const { sResponse, rrRequestId } = await readRr(inspectorConsumer);
+  return { sResponse, sRandomWord, rrRequestId, vrfRequestId };
+}
+
+export async function readVrf(inspectorConsumer: Contract) {
+  const sRandomWord = await inspectorConsumer.sRandomWord();
+  const vrfRequestId = await inspectorConsumer.vrfRequestId();
+  return { sRandomWord, vrfRequestId };
+}
+export async function readRr(inspectorConsumer: Contract) {
+  const sResponse = await inspectorConsumer.sResponse();
+  const rrRequestId = await inspectorConsumer.rrRequestId();
+  return { sResponse, rrRequestId };
+}
+
+export async function requestAll(
+  ACC_ID: string,
+  explorerBaseUrl: string,
+  inspectorConsumer: Contract,
+  networkName: string
+) {
+  await requestVrf(ACC_ID, explorerBaseUrl, inspectorConsumer, networkName);
+  await requestRr(ACC_ID, explorerBaseUrl, inspectorConsumer);
+}
+
+export async function requestVrf(
+  ACC_ID: string,
```

Comment:
Same comment as above 🙌 

```suggestion
  accId: string,
```